lib/module_importer.js

```diff
@@ -105,24 +103,15 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
 
   return function(uri, prev, done) {
     var isRealFile = fs.existsSync(prev);
-    var match = uri.match(IMPORT_REGEX);
-
-    if (match) {
-      // This is an import of the form "<node_module>/some/file"
-      var moduleName = match[1];
-      var relativePath = match[2];
-      var pkgRootDir = isRealFile ? packageRootDir(path.dirname(prev)) : root;
-      var jsFile = getModuleByName(moduleName, pkgRootDir);
-
-      if (!jsFile) {
-        console.error("No Eyeglass module named '" + moduleName +
```

Comment:
We can, but we won't know if they intended the import as a node style or not. Still, I think that any error is better than none and will add the suggestion and rebase :)



—
Jakob Heuser

On Fri, Mar 20, 2015 at 12:43 PM, Edilberto Ruvalcaba
<notifications@github.com> wrote:

>> @@ -105,24 +103,15 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
>>  
>>    return function(uri, prev, done) {
>>      var isRealFile = fs.existsSync(prev);
>> -    var match = uri.match(IMPORT_REGEX);
>> -
>> -    if (match) {
>> -      // This is an import of the form "<node_module>/some/file"
>> -      var moduleName = match[1];
>> -      var relativePath = match[2];
>> -      var pkgRootDir = isRealFile ? packageRootDir(path.dirname(prev)) : root;
>> -      var jsFile = getModuleByName(moduleName, pkgRootDir);
>> -
>> -      if (!jsFile) {
>> -        console.error("No Eyeglass module named '" + moduleName +
> @Jakobo shouldn't we be able to show this error in the last case of this [if-else block](https://github.com/sass-eyeglass/eyeglass/blob/master/lib/module_importer.js#L167)? 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/sass-eyeglass/eyeglass/pull/27/files#r26870192